backend/lib/server_code/api/roomRoutes.js

```diff
@@ -14,24 +19,40 @@ roomRouter.use((req, res, next) => {
   next();
 });
 
-roomRouter.post("/create", (req, res) => {
+roomRouter.post("/create", async (req, res) => {
   const { creatorID } = req.body;
-  //roomPlayers.push(creatorID);
+  const roomID = roomIDCreator();
   roomCreated = true;
-  res.send({ roomID: 1, creatorID, creationDate: res.date });
+  let [socket, socketPort] = [null, null];
```

Comment:
why u init these parameters here and not in the try?

Reply:
actually i dont need to init them at all

Reply:
b1a1010c0b7a17960fac9f061dced9c6007a7785

Reply:
b1a1010

Reply:
b1a1010c0b7a17960fac9f061dced9c6007a7785